tests/integration-tests/tests/scaling/test_scaling/test_scaling_special_cases/include_custom_partition_large.sh.jinja

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -e
```

Comment:
This is very cryptic.

Can you add some comments here to explain which are the "design decisions" behind these custom partitions ?

Can we somehow "code" them ?
If we have to change something in the future ... manually checking if a specific ID in [1:1600] is in one partition or in another will be difficult

Reply:
Let's sync on this.